.eslintignore

```diff
@@ -19,17 +19,19 @@ docs/api
 docs/build
 examples/example.spec.ts
 examples/federated/core_package/index.template.js
+examples/federated/core_package/index.js
+examples/federated/labextensions
+galata/playwright-report
 jupyterlab/chrome-test.js
 jupyterlab/geckodriver
+jupyterlab/staging/yarn.js
+jupyterlab/staging/index.js
+jupyterlab/staging/webpack.config.js
```

Comment:
Looks like we also need to add them to `.prettierignore`.

Reply:
Good catch - I updated the PR; remaining CI errors are not related.